src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  font-family: "Roboto" , sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  max-width: 100%;
+  padding: 0 50px;
+}
+
+.logo {
+  padding-top: 5px;
+}
+
+.menu_list {
+  display: flex;
+}
+
+.nav_item {
+  display: flex;
+  padding-left: 20px;
+}
+
+.nav_item :first-child {
+  padding-left: 0;
+}
+
+.nav_link {
+  padding: 23px 0;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 1s;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.nav_1 {
+  color: #00acdc;
+}
+
+.nav_link.is_active {
+  position: relative;
+}
+
+.is_active.nav_link {
+  color: #00acdc;
+}
```

you have same selectors sot you can merge them into one
```suggestion
.nav_link.is_active {
  position: relative;
  color: #00acdc;
}
```